docs/cli/protocol.md

```
@@ -0,0 +1,431 @@
+# Output as JSON
+galaxy protocol liquidity info XLM USDC --json
+```
+
+---
+
+## Common Options
+
+All protocol commands support these common options:
+
+| Option | Description |
+|--------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)